bin/upload

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
```

Comment:
like above executable, add comments here describing how this fits into workflow? And document all of this in README?  But this is awesome improvements to the process !! 👍